wandb/env.py

```diff
@@ -409,3 +410,12 @@ def disable_git(env: Optional[Env] = None) -> bool:
     if isinstance(val, str):
         val = False if val.lower() == "false" else True
     return val
+
+
+def disable_job_creation(env: Optional[Env] = None) -> bool:
+    if env is None:
+        env = os.environ
+    val = env.get(DISABLE_JOB_CREATION, default="False")
+    if isinstance(val, str):
+        val = False if val.lower() == "false" else True
```

Comment:
technically, you don't need this part `_str_as_bool` in settings will normalize it.